### DISCUSSION TITLE
Is it possible for us to use MSP to control the iNav as RC remoteControl do?

### DISCUSSION THREAD

**Discussion by lida2003** [iNavFlight/inav#8537] [Q&A] [answered]
Is it possible for us to use MSP to control the iNav as RC remoteControl do?

I mean use MSP-GCS software(lightweight) to arm/disarm/throttle/yaw/pitch/roll remotely via 433MHz/ZigBee(what ever) hardware module?

Any advice and suggestion is well appreciated.

PS: it seems marvlink of inav is NOT that strong as MSP protocol, am i right~~~~

**Comment by stronnag** [comment, marked as answer]
It is. There's an example (and some caveats) at https://github.com/stronnag/msp_set_rx

**Reply by lida2003** [reply]
That's great！ Thanks.
